Fix Javadoc warnings

### Describe the bug

There are javadoc warnings

### Expected behavior

_No response_

### Minimal reproducible example

_No response_

### Add-on Version

2.0.0

### Vaadin Version

-

### Additional information

```
[WARNING] Javadoc Warnings
[WARNING] J:\fc\addons\WhatsappButton\src\main\java\com\flowingcode\vaadin\addons\whatsappbutton\WhatsappButton.java:102: warning: @param argument "invite" is not a parameter name.
[WARNING] * @param invite code the invite code to set
[WARNING] ^
[WARNING] J:\fc\addons\WhatsappButton\src\main\java\com\flowingcode\vaadin\addons\whatsappbutton\WhatsappButton.java:174: warning: @param argument "redirect" is not a parameter name.
[WARNING] * @param redirect true to enable redirect
[WARNING] ^
[WARNING] J:\fc\addons\WhatsappButton\src\main\java\com\flowingcode\vaadin\addons\whatsappbutton\WhatsappButton.java:193: warning: @param argument "redirect" is not a parameter name.
[WARNING] * @param redirect true to enable redirect
[WARNING] ^

--- a/pom.xml
+++ b/pom.xml
@@ -304,7 +304,7 @@
                         <configuration>
                             <quiet>true</quiet>
                             <doclint>none</doclint>
-                            <failOnWarnings>false</failOnWarnings>
+                            <failOnWarnings>true</failOnWarnings>
                             <links>
                                 <link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>
                             </links>

--- a/src/main/java/com/flowingcode/vaadin/addons/whatsappbutton/WhatsappButton.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/whatsappbutton/WhatsappButton.java
@@ -99,7 +99,7 @@ public class WhatsappButton extends Component {
     /**
      * Sets an invite code.
      * 
-     * @param invite code the invite code to set
+     * @param inviteCode the invite code to set
      */
     public void setInviteCode(String inviteCode) {
         this.getElement().setProperty("invitecode", inviteCode);
@@ -171,7 +171,7 @@ public class WhatsappButton extends Component {
     /**
      * Sets responsive option.
      * 
-     * @param redirect true to enable redirect
+     * @param responsive true to enable responsive option
      */
     public void setResponsive(boolean responsive) {
         this.getElement().setProperty("responsive", responsive);
@@ -190,7 +190,7 @@ public class WhatsappButton extends Component {
      * Sets dialog option. If true, a dialog will show on button click indicating whatsapp is
      * opening.
      * 
-     * @param redirect true to enable redirect
+     * @param dialog true to enable dialog option
      */
     public void setShowDialog(boolean dialog) {
         this.getElement().setProperty("dialog", dialog);

